lblod/frontend-embeddable-notule-editor
# GN-4723: addition of floating table menu

## Overview
This PR adds support for a floating table menu which provides users quick access to table editing tools.
This floating menu is implemented to be as non-intrusive as possible: it only displays when clicking; and disappears when typing.
The menu contains the following actions:
- Add row after
- Add row before
- Add column after
- Add column before
- Toggle row header
- Toggle column header
- Delete row
- Delete column
- Delete table

##### connected issues and PRs:
[GN-4723](https://binnenland.atlassian.net/browse/GN-4723?atlOrigin=eyJpIjoiMmVjNzhmNTc2ZDZhNGI2M2FkNGZmNWVkZDU2MjM1NDgiLCJwIjoiaiJ9)
https://github.com/lblod/ember-rdfa-editor/pull/1144

### How to test/reproduce
- `npm start`
- open the test-app
- insert a table
- click inside the table => the floating menu should appear
- test if you can access/use all the tools correctly
- type inside the table => the menu should disappear


### Checks PR readiness
